testFixtures/org/jetbrains/java/decompiler/SingleClassesTestBase.java

```diff
@@ -247,14 +268,14 @@ public void run(Object[] options, SingleClassesTestBase base) throws IOException
           throw new UncheckedIOException(e);
         }
         //noinspection ConstantConditions
-        assumeTrue(false, referenceFile.getFileName() + " was not present yet");
+        abort(referenceFile.getFileName() + " was not present yet");
```

Comment:
What's the reasoning behind this change?

Reply:
The reasoning was to follow the `fail` code I had above, but I've reverted that with my new commit